lib/rentlinx/modules/photoable.rb

```diff
@@ -0,0 +1,27 @@
+module Rentlinx
+  module Photoable
```

Comment:
Ahh, this is probably the right place for the <code>get_photos_for_property_id</code> method (see https://github.com/appfolio/rentlinx_client/pull/20/files#r35021434).  It could probably be renamed to just <code>get_photos</code> and be called without a parameter, since both types of photoables have a property_id attribute.